classes/shop/Shop.php

```diff
@@ -521,7 +521,7 @@ public function getBaseURI()
      *
      * @return string|bool complete base url of current shop
      */
-    public function getBaseURL($auto_secure_mode = false, $add_base_uri = true)
+    public function getBaseURL($auto_secure_mode = true, $add_base_uri = true)
```

Comment:
Should we check if `PS_SSL_ENABLED` is enabled ?

Reply:
There is two ways to get url sheme in prestashop
- check PS_SSL_ENABLED 
- Tools::usingSecureMode : that check if https is activated in server envs

In Link::getBaseLink, http or https is checked against PS_SSL_ENABLED and PS_SSL_ENABLED_EVERYWHERE

This is not the point of my pr to change this two ways.

Reply:
so even if `PS_SSL_ENABLED` is disabled the `Shop::getBaseURL` will always use secure mode by default?

Reply:
No, Shop::getBaseURL will check is the context is in https or http by defaut.